app/models/links/link.rb

```diff
@@ -3,15 +3,15 @@
 class Links
   class Link
     include ActionView::Helpers::TagHelper
+    include IndexLinks::Structures
```

Comment:
These behaviors only apply to IndexLinks, and not to MarcLinks or EdsLinks, so it seems to me this should say in IndexLinks.

Reply:
I don't understand what you are saying. Are you saying finding_aid? should go into a different concern?